src/main/java/uk/gov/register/resources/v2/BlobResource.java

```diff
@@ -47,13 +47,13 @@ public ItemResource(RegisterReadOnly register, ViewFactory viewFactory) {
             ExtraMediaType.APPLICATION_SPREADSHEET
     })
     @Timed
-    public ItemView getItemDataByHex(@PathParam("item-hash") String itemHash) throws FieldConversionException {
-        return getItem(itemHash).map(viewFactory::getItemMediaView)
-                .orElseThrow(() -> new NotFoundException("No item found with item hash: " + itemHash));
+    public ItemView getBlobDataByHex(@PathParam("blob-hash") String blobHash) throws FieldConversionException {
+        return getBlob(blobHash).map(viewFactory::getItemMediaView)
+                .orElseThrow(() -> new NotFoundException("No item found with item hash: " + blobHash));
```

Comment:
should this text be _No item found with blob hash:_?

Reply:
Yes it should - I think there are a few of these messages that need changing